pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -124381,15 +124381,15 @@ self: {
        sha256 = "0lm52xsgf3sayj5d32fyf9fy89zinn7c4z6rq4qw2bsnsdw8hcyb";
        buildDepends = [ base bytestring cairo glib gtk3 mtl pango text ];
        buildTools = [ gtk2hs-buildtools ];
-       pkgconfigDepends = [ webkit ];
+       pkgconfigDepends = [ webkitgtk24x ];
        homepage = "http://projects.haskell.org/gtk2hs/";
        description = "Binding to the Webkit library";
        license = stdenv.lib.licenses.lgpl21;
        hydraPlatforms = stdenv.lib.platforms.none;
-     }) { inherit (pkgs) webkit;};
+     }) { inherit (pkgs) webkitgtk24x;};
```

Comment:
Same here; please override `webkit` with the proper name in the common configuration.